server/sonar-server/src/main/java/org/sonar/server/component/ws/ComponentDtoToWsComponent.java

```diff
@@ -57,6 +57,9 @@ private ComponentDtoToWsComponent() {
     setNullable(emptyToNull(dto.language()), wsComponent::setLanguage);
```

Comment:
commit title "SONAR-9096 add visibility flag to response: api/components/search_projects" is too long

"SONAR-9096 add visibility to response api/components/search_projects" should work (69 chars long)